benchmark_runner/common/clouds/IBM/assisted_installer_latest_version.py

```diff
@@ -0,0 +1,61 @@
+import json
+
+# import urllib library
+from urllib.request import urlopen
+
+
+class OCPVersions:
+    """
+    This class get the latest assisted installer version from https://openshift-release.apps.ci.l2s4.p1.openshiftapps.com/graph
+    """
+    # MAJOR.MINOR.PATCH.BUILD
+    MAJOR = 0
+    MINOR = 1
+    PATCH = 2
+    BUILD = 3
+
+    def __init__(self):
+        self.__url = "https://openshift-release.apps.ci.l2s4.p1.openshiftapps.com/graph"
+
+    def __get_versions(self):
+        """
+        This method get the OCP versions from url and return json data
+        :return: versions in json
+        """
+        # store the response of URL
+        response = urlopen(self.__url)
+
+        # storing the JSON response
+        # from url in data
+        return json.loads(response.read())
+
+    def get_latest_version(self, latest_version: str):
+        """
+        This method get the latest version from json data
+        :param latest_version:
+        :return:
+        """
+        patch_number_list = []
+        build_number_list = []
+        patch_version = ''
+        build_version = ''
+
+        for version in self.__get_versions()['nodes']:
+            version_data = str(version['version']).split('.')
+            if latest_version == f'{version_data[self.MAJOR]}.{version_data[self.MINOR]}':
+                if len(version_data) == self.PATCH + 1:
+                    patch_version = f'{version_data[self.MAJOR]}.{version_data[self.MINOR]}'
+                    patch_number_list.append(int(version_data[self.PATCH]))
+                if len(version_data) == self.BUILD + 1 and len(version_data[self.BUILD]) <= 2:
```

Comment:
Not sure whether you should use `len(version_data) == self.I_BUILD + 1` or `len(version_data > self.I_BUILD`.  This is a bit of a nit; just want to make sure that it's obvious what's going on.